datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -176,20 +176,62 @@ private fun Questionnaire.QuestionnaireItemComponent.createQuestionnaireResponse
   QuestionnaireResponse.QuestionnaireResponseItemComponent {
   return QuestionnaireResponse.QuestionnaireResponseItemComponent().apply {
     linkId = this@createQuestionnaireResponseItem.linkId
-    if (this@createQuestionnaireResponseItem.initial.isNotEmpty()) {
-      answer =
-        mutableListOf(
-          QuestionnaireResponse.QuestionnaireResponseItemAnswerComponent().apply {
-            value = this@createQuestionnaireResponseItem.initial[0].value
-          }
-        )
-    }
+    answer = createQuestionnaireResponseWithInitialItem()
     this@createQuestionnaireResponseItem.item.forEach {
       this.addItem(it.createQuestionnaireResponseItem())
     }
   }
 }
 
+private fun Questionnaire.QuestionnaireItemComponent.createQuestionnaireResponseWithInitialItem():
+  MutableList<QuestionnaireResponse.QuestionnaireResponseItemAnswerComponent>? {
+  if (this@createQuestionnaireResponseWithInitialItem.initial.isNotEmpty()) {
```

Comment:
do you need this at all? 
```suggestion
  if (initial.isNotEmpty()) {
```

Reply:
Yes and done